File: server/app/routers/fetch.js
const express = require("express");
const mongoose = require("mongoose");

// import mongoose schemas
const Team = require("../models/team");
const Card = require("../models/card");

mongoose.connect("mongodb://localhost:27017/wind");

const router = express.Router();
router.post("/fetch", async (req, res) => {
	res.header("Access-Control-Allow-Origin", "*");

	const teams = await Team.find({});
	res.status(200).json({ result: teams });
});


module.exports = router;

File: fetch.py
import requests
import json
from datetime import datetime
from time import time

#BASE_URL = 'http://localhost:12345'
BASE_URL = 'http://140.112.174.222:' + str(input())
API_AUTH = '/wind/fetch'

session = requests.Session()

print(BASE_URL+API_AUTH)

req = session.post(
    BASE_URL + API_AUTH, json={},
    allow_redirects=True
)

print(req)

